hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/TestSequenceFileSerialization.java

```diff
@@ -49,6 +49,7 @@ public void tearDown() throws Exception {
     fs.close();
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
I'd leave ths out (or make a PR for trunk on its own and TestSequenceFileSync)

Reply:
This should be fixed now after cleaning the commit and revert part in hadoop-common.